ea-factories/src/factories/balance.ts

```diff
@@ -88,12 +97,13 @@ export const make: ExecuteFactory<BalanceConfig> = (config) => async (input) =>
     toValidAccount(jobRunID, acc, config),
   )
   const accountsToProcess = accounts.filter((acc) => !acc.warning)
-
-  const getBalances = config.getBalances || toGetBalances(config.getBalance)
+  const getBalances = config.getBalances || toGetBalances<C>(config.getBalance)
 
   const key = (acc: Account) => `${acc.coin}-${acc.chain}`
   const groups = Array.from(util.groupBy(accountsToProcess, key).values())
-  const requests = groups.flatMap((group) => getBalances(group as Account[], config))
+  const requests = groups.flatMap((group) => {
+    return getBalances(group as Account[], config)
+  })
```

Comment:
How do you feel about one-lining this?
```suggestion
  const requests = groups.flatMap((group) => 
    getBalances(group as Account[], config)
  )
```
```suggestion
  const requests = groups.flatMap((group) => {
    return getBalances(group as Account[], config)
  })
```